src/Components/Patient/ManagePatients.tsx

```diff
@@ -723,6 +725,9 @@ export const PatientManager = () => {
     };
   };
 
+  const onlyAccessibleFacility =
+    permittedFacilities?.count === 1 ? permittedFacilities.results[0] : null;
```

Comment:
This will always be true, right? since we're limiting it to 1 above

Reply:
No, `results.length` will always be equal to 1 since we've set a `limit` to 1. But `count` would always be the no. of elements user has access to regardless of the current page right.